lib/elixir/lib/inspect/algebra.ex

```diff
@@ -588,6 +588,21 @@ defmodule Inspect.Algebra do
     doc_cons(doc1, doc2)
   end
 
+  @doc ~S"""
+  Prevents breaks inside the document.
```

Comment:
```suggestion
  Disable any rendering limit while rendering the given document.
```

I think this makes more sense. Because if you have an explicit break, it is still respected.